tests/unit/test_charm.py

```diff
@@ -334,3 +334,11 @@ def test_given_certificate_is_stored_when_on_certificates_relation_broken_then_c
 
         with pytest.raises(SecretNotFoundError):
             self.harness._backend.secret_get(label="certificate")
+
+    def test_given_csr_is_not_stored_when__request_certificate_called_then_runtime_error_is_not_raised(  # noqa: E501
```

Comment:
```suggestion
    def test_given_csr_is_not_stored_when_request_certificate_called_then_runtime_error_is_not_raised(  # noqa: E501
```

Really sorry for the netpicking :D

Reply:
Method name was  _request_certificate so I just kept the underscore as it is.

Reply:
I will remove _ to follow the same convention.